site/components/DeleteConfirmationPopup.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { cleanup, fireEvent, render } from "@testing-library/react";
+import { describe, it, expect, afterEach } from "vitest";
+import DeleteConfirmationPopup from "./DeleteConfirmationPopup";
+
+describe("DeleteConfirmationPopup", () => {
+    it("should render correctly", () => {
+        const { container } = render(
+            <DeleteConfirmationPopup
+                entityName="test"
+                deleteUrl="https://test.com/api/delete"
+                cancelActionHandler={() => ""}
```

Comment:
Should we be using vi.fn() here? I think most tests do that instead of these empty functions.

Reply:
Also in the rest of the file

Reply:
done